app/helpers/agent_helper.rb

```diff
@@ -6,6 +6,14 @@ def agent_show_view(agent)
     end
   end
 
+  def toggle_disabled_text
+    if cookies[:huginn_view_only_enabled_agents]
+      "Show Disabled Agents"
```

Comment:
One minor thing, can you add a leading space like it is done for the other links? The text looks a little bit cramped right next to the icon.